sdk/bom/ci.yml

```diff
@@ -0,0 +1,43 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+resources:
```

Comment:
Should this file go under `eng/pipelines/`?

Reply:
Nope, this is akin to the other `ci.yml` files that exist under all the module root directories.